solidity/dashboard/src/components/Web3Status.jsx

```diff
@@ -26,8 +26,10 @@ export const Web3Status = () => {
           <div className={`wallet__address${isActive ? "--active" : ""}`}>
             {isActive ? shortenAddress(yourAddress) : "connect wallet"}
           </div>
-          <div className="wallet__menu">
-            {isActive ? <WalletMenu /> : <WalletOptions />}
+          <div className="wallet__menu-container">
+            <div className="wallet__menu">
+              {isActive ? <WalletMenu /> : <WalletOptions />}
+            </div>
```

Comment:
Just a quick note why I had to move the wallet__menu into a container.

![image](https://user-images.githubusercontent.com/40306834/103708065-18921f80-4fb0-11eb-964c-4a163477c1ef.png)

The main issue with the wallet menu was that it was showing when we hovered on the list of the items, while they were invisible. I changed it so that the menu list appears only when you hover on `CONNECT WALLET` button and we are keeping the list visible while we hover on it (but only when we hovered on the `CONNECT WALLET` button first).

The reason i moved the wallet__menu to a container is because of the small gap between the CONNECT WALLET button and the menu list, which caused the menu list to disappear when we wanted to move the cursor from the button to the list through that gap. Now this 'gap' is a container, which allowed me to keep it visible with proper styling.